Modules/Filtering/ImageIntensity/include/itkSymmetricEigenAnalysisImageFilter.h

```diff
@@ -106,6 +106,12 @@ class SymmetricEigenAnalysisFunction
       m_Calculator.SetOrderEigenValues(false);
     }
   }
+  void
+  SetOrderEigenValuesBy(EigenValueOrderEnum order)
+  {
+    this->OrderEigenValuesBy(order);
+  }
+
```

Comment:
I do not know what motivates adding the `SetOrderEigenValuesBy` method as a synonym of the `OrderEigenValuesBy` method of the `SymmetricEigenAnalysisFunction` class, so I do not know if the same should be added for the other classes in this file ... or elsewhere in ITK:
```shell
$ fgrep -Rl OrderEigenValuesBy *
Modules/Filtering/ImageFeature/include/itkHessian3DToVesselnessMeasureImageFilter.hxx
Modules/Filtering/ImageIntensity/include/itkSymmetricEigenAnalysisImageFilter.h
Modules/Filtering/ImageIntensity/test/itkSymmetricEigenAnalysisImageFilterTest.cxx
```

Reply:
Good idea to check. Looking into the details, the call in `itk::Hessian3DToVesselnessMeasureImageFilter` is internal, instead of the interface.